platform/broadcom/saibcm-modules/sdklt/linux/bde/ngbde_dma.c

```diff
@@ -35,6 +35,9 @@ MODULE_PARM_DESC(dma_debug,
 /*! Default number of DMA memory pools per device. */
 #define NUM_DMAPOOL_DEFAULT     1
 
+/*! Define the macro to the integer constant. */
```

Comment:
>Define the macro to the integer constant [](http://example.com/codeflow?start=4&length=40)

The comment does not help reader. You may remove or refine the words.

Reply:
Done.